src/shared/copy.c

```diff
@@ -88,6 +88,23 @@ static int fd_is_nonblock_pipe(int fd) {
         return FLAGS_SET(flags, O_NONBLOCK) ? FD_IS_NONBLOCKING_PIPE : FD_IS_BLOCKING_PIPE;
 }
 
+static int look_for_signals(CopyFlags copy_flags) {
+        int r;
+
+        if ((copy_flags & (COPY_SIGINT|COPY_SIGTERM)) == 0)
```

Comment:
`FLAGS_SET`?

Reply:
FLAG_SET check would be different: it checks *all* mentioned bits are set. here we want to check if neither is set

Reply:
I was thinking more about `FLAGS_SET(copy_flags, COPY_SIGINT|COPY_SIGTERM)` which is a pattern used in a few places already - but it's not hugely important

Reply:
but that checks if both flags are set. here we want to check that neither is set. It's a different test.

`FLAGS_SET(x, f)` checks `(x & f) == f`

but here we need:

`FLAGS_SET(x, f)` checks `(x & f) != 0)`